app/src/main/java/fr/free/nrw/commons/profile/ProfileActivity.java

```diff
@@ -140,14 +149,13 @@ private void setTabs() {
         fragmentList.add(leaderboardFragment);
         titleList.add(getResources().getString(R.string.leaderboard_tab_title).toUpperCase());
 
-        if (shouldShowContributions) {
```

Comment:
In the past, why was this `if (shouldShowContributions) {` added? You can use `git blame` to find out.